pkg/tasks/prometheusoperator_user_workload.go

```diff
@@ -117,6 +117,15 @@ func (t *PrometheusOperatorUserWorkloadTask) create(ctx context.Context) error {
 		return errors.Wrap(err, "waiting for Prometheus Operator CRs to become available failed")
 	}
 
+	pr, err := t.factory.PrometheusOperatorUserWorkloadPrometheusRule()
```

Comment:
Should we change the destroy function to delete the rules when UWM is disabled?

Reply:
Done